tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/LanguageExtensionsSuite.scala

```diff
@@ -0,0 +1,41 @@
+package scala.meta.tests.parsers.dotty
+
+import scala.meta._
+
+class LanguageExtensionsSuite extends BaseDottySuite {
```

Comment:
i don't know if we need a separate suite for just one modifier and 4 tests. isn't there already a suitable one?

Reply:
I created a new suite, so that we can put all the other new language extension features there. Is that ok, or should I move it somewhere?
There are quite a few extensions missing here, so the test suite can fill up quickly.

Reply:
would you like to add these tests, too? 😁

Reply:
I meant the ones that aren't implemented here yet.

e.g.
- `into`
- `namedTuples`
- `captureChecking`
- `namedTypeArguments`
- etc.